packages/idos-sdk-js/src/lib/grants/grants.ts

```diff
@@ -74,7 +74,7 @@ export class Grants {
     accountId?: string;
     publicKey?: string;
   }): Promise<ConnectedGrants> {
-    let child;
+    let child: EvmGrants | NearGrants | never;
```

Comment:
nit: maybe we can narrow the type a bit?
```suggestion
    let child: EvmGrants | NearGrants;
```